### ISSUE TITLE
Do not reconnect when queryParams value changed.

### ISSUE THREAD

**Issue by watanabeshuji** [robtaussig/react-use-websocket#53]
Hi, I use useWebSocket with queryParams.
And I need to change parameter on user interface (user_id is query param).

When url contains variable params, it works.
But when using queryParams, it does not work.

Work
```
const [userId, setUserId] = useState(1)
useWebSocket(`wss://hostname/?user_id={userId}`, {
})
```

Not Work!
```
const [userId, setUserId] = useState(1)
useWebSocket(`wss://hostname/`, {
    queryParams: {
      user_id: userId,
    },
})
```

**Comment by robtaussig** [comment]
Excellent point -- it should definitely reconnect when changing queryParams. Will fix immediately.

**Comment by robtaussig** [comment]
Changes to queryParams should now trigger the same reconnect behavior as if changing the url:

2401015

Released as version 2.1.0
